pkg/storegateway/bucket_test.go

```diff
@@ -2879,13 +2823,13 @@ func labelNamesFromSeriesSet(series []*storepb.Series) []string {
 		}
 	}
 
-	labels := make([]string, 0, len(labelsMap))
+	ll := make([]string, 0, len(labelsMap))
```

Comment:
what's wrong with `labels` 😄

Reply:
I think, my IDE's complained that `labels` overlaps with imports of the same name, and I've just yolo'ed it. Looking through the changes, I don't think it belongs here. I've reverted it back.